src/main/java/org/opensearch/ad/ml/ThresholdingResult.java

```diff
@@ -21,9 +24,20 @@
     private final double grade;
     private final double confidence;
     private final double rcfScore;
+    private long totalUpdates;
+    private boolean startOfAnomaly;
+    private boolean inHighScoreRegion;
+    private int relativeIndex;
+    private double[] currentTimeAttribution;
```

Comment:
After reading more code, I feel this field named as "currentTimexxx" as the attribution may be not for current time, it may be for some oldValues ? So you name it "currentTimexxx" to indicate the attribution is for current interval aggregated data point, not for some old values?